jekyll/_cci2/language-java-maven.md

```diff
@@ -165,7 +165,7 @@ version: 2.0
 jobs:
   build:
     docker:
-      - image: circleci/openjdk:stretch
+      - image: cimg/openjdk.17.0.1
```

Comment:
Typo

`cimg/openjdk.17.0.1` should be `cimg/openjdk:17.0.1`

Reply:
Fixed